src/Client.php

```diff
@@ -141,6 +141,16 @@ public function create(AutotaskObjects\Entity $obj)
         $params = new AutotaskObjects\CreateParam($obj);
         return $this->_call('create', array($params));
     }
+    
+    public function bulkCreate(array $objs)
+    {
+        $createObjs;
+        foreach ($objs as $obj) {
+           if (!$createObjs) $createObjs = new AutotaskObjects\CreateParam($obj);
```

Comment:
The logic here looks different than the `bulkUpdate` -- I assume that is due to copy/pasting my `bulkUpdate` function, and then finding issues with `bulkUpdate`.

Can you verify that `bulkCreate` works as expected?